python/catboost.Dockerfile

```
@@ -0,0 +1,71 @@
+ARG PYTHON_VERSION=3.11
+ARG BASE_IMAGE=python:${PYTHON_VERSION}-slim-bookworm
+ARG VENV_PATH=/prod_venv
+
+FROM ${BASE_IMAGE} AS builder
+
+# Install necessary dependencies for building Python packages
+RUN apt-get update && apt-get install -y --no-install-recommends python3-dev curl build-essential && apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+# Install uv
+RUN curl -LsSf https://astral.sh/uv/install.sh | sh && \
+    ln -s /root/.local/bin/uv /usr/local/bin/uv
+
+# Activate virtual env
+ARG VENV_PATH
+ENV VIRTUAL_ENV=${VENV_PATH}
+RUN python3 -m venv ${VIRTUAL_ENV}
+ENV PATH="$VIRTUAL_ENV/bin:$PATH"
+
+# Copy and install dependencies for kserve using uv
+COPY kserve/pyproject.toml kserve/uv.lock kserve/
+RUN cd kserve && uv sync --active --no-cache
+COPY kserve kserve
+RUN cd kserve && uv sync --active --no-cache
+
+# Copy and install dependencies for kserve-storage using uv
+COPY storage/pyproject.toml storage/uv.lock storage/
+RUN cd storage && uv sync --active --no-cache
+
+COPY storage storage 
+RUN cd storage && uv pip install . --no-cache
+
+# Copy and install dependencies for catboostserver using uv
+COPY catboostserver/pyproject.toml catboostserver/
+RUN cd catboostserver && uv pip install catboost~=1.2.7 --no-cache
+COPY catboostserver catboostserver
+RUN cd catboostserver && uv pip install . --no-cache
```

```suggestion
RUN cd catboostserver && uv sync --active --no-cache
```

For the sake of consistency